test/unexpected-knex.spec.js

```diff
@@ -1092,8 +1092,9 @@ describe('unexpected-knex', function() {
             ),
             'to be rejected with',
             dontIndent`
-                expected 'select * from "foo"'
-                to have rows satisfying function ( /*...*/ ) { /*...*/ }
+                expected 'select * from "foo"' to have rows satisfying
+                rows =>
+                expect(rows, 'to equal', [])
```

Comment:
@papandreou @sunesimonsen @alexjeffburke seems like there might be a bug in the outputting of arrow functions here. compare with the other output on [line 1062](https://github.com/unexpectedjs/unexpected-knex/pull/24/files#diff-e380c8bb17419a470f5eaa1429e09d35R1062). Should I create an issue?

Reply:
https://github.com/unexpectedjs/unexpected/issues/549